plugins/backend/totp/backend.go

```diff
@@ -0,0 +1,45 @@
+package totp
```

Comment:
Do we want to include another variant of the TOTP backend just for testing. Can the main function for the test just import the builtin version of this backend?